Project.toml

```diff
@@ -17,6 +17,8 @@ ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 
 [compat]
 ADNLPModels = "0.3.1"
```

Comment:
Is ADNLPModels really a dependency here?

Reply:
I use it when I define the `get_expr_tree` function for `ADNLPModel`.
Maybe it can be change by `AbstractNLPModel` since it does not use `ADNLPModel` methods.

Reply:
Yes.

Reply:
I will specify the dependency for the v0.2.0 with Registers #32.